config/deploy/production.rb

```diff
@@ -1,3 +1,6 @@
+# We only want to update crontab in production
+require 'whenever/capistrano'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.